core/trino-main/src/test/java/io/trino/execution/TestMemoryRevokingScheduler.java

```diff
@@ -107,8 +108,12 @@ public void tearDown()
     {
         queryContexts.clear();
         memoryPool = null;
+        taskExecutor.stop();
```

Comment:
should we use closer here to make sure all cleanup routines are called?

Reply:
I don't see a need to. `@AfterClass` failure is a failure for tests execution so we won't ignore it.